core/src/error/mod.rs

```diff
@@ -47,9 +47,9 @@ pub type Result<T> = ::std::result::Result<T, Error>;
 ///    This preserves all span information, suggestions, etc. Wrapping a `darling::Error` in
 ///    a custom error enum works as-expected and does not force any loss of fidelity.
 /// 2. Do not use early return (e.g. the `?` operator) for custom validations. Instead,
-///    create a local `Vec` to collect errors as they are encountered and then use
-///    `darling::Error::multiple` to create an error containing all those issues if the list
-///    is non-empty after validation. This can create very complex custom validation functions;
+///    create an [`error::Collector`](Collector) to collect errors as they are encountered.  Then use
```

Comment:
I kind of like the name `Accumulator` or `ErrorAccumulator` for this, given that `collect` has strong meanings to Rust.

Reply:
Makes sense.